docs/api/zh/geometries/BoxGeometry.html

```diff
@@ -4,14 +4,19 @@
 		<meta charset="utf-8" />
 		<base href="../../../" />
 		<script src="page.js"></script>
-		<link type="text/css" rel="stylesheet" href="page.css" />
+    <link
+      type="text/css"
+      rel="stylesheet"
+      href="page.css" />
```

Comment:
Please make sure to use consistent tabs rather than spaces through the changed lines.